app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/PTEnrolmentOnAnotherAccount.scala.html

```diff
@@ -16,47 +16,79 @@
 
 @import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.routes
 @import uk.gov.hmrc.govukfrontend.views.html.components._
-@import uk.gov.hmrc.taxenrolmentassignmentfrontend.models.AccountDetails
 @import uk.gov.hmrc.taxenrolmentassignmentfrontend.config.AppConfig
 @import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 @import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.helpers.CreateSummaryRow
 @import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.Layout
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.models._
 
 @this(
 layout: Layout,
 govukSummaryList : GovukSummaryList,
-summaryRow: CreateSummaryRow
+summaryRow: CreateSummaryRow,
+govukInsetText : GovukInsetText
 )
 
-@(ptAccountDetails: AccountDetails, currentAccountHasSA: Boolean)(implicit request: Request[AnyRef], messages: Messages, appConfig: AppConfig)
+@(ptEnrolmentOtherAccountViewModel: PTEnrolmentOtherAccountViewModel)(implicit request: Request[AnyRef], messages: Messages, appConfig: AppConfig)
+
+@content = {
+    <a class = "govuk-link" href=@routes.ReportSuspiciousIDController.viewNoSA>@messages("ptEnrolmentOnOtherAccount.linkText")</a>
+}
 
 @layout(
     pageTitle = Some(messages("ptEnrolmentOnOtherAccount.title")), None) {
 
-    <h2 class="govuk-heading-xl">@messages("ptEnrolmentOnOtherAccount.heading")</h2>
+    <h1 class="govuk-heading-xl">@messages("ptEnrolmentOnOtherAccount.heading")</h1>
+
+    <p class="govuk-body">@messages("ptEnrolmentOnOtherAccount.paragraph1", ptEnrolmentOtherAccountViewModel.ptAccountDetails.userId)</p>
+    <p class="govuk-body">@messages("ptEnrolmentOnOtherAccount.paragraph2", ptEnrolmentOtherAccountViewModel.currentAccountDetails.userId)</p>
+
```

Comment:
This appears on a new line whereas in the design it's still on one line.
![image](https://user-images.githubusercontent.com/29196699/166441999-3fbe009e-46c5-444a-bde3-56997cc3dda2.png)

in comparison to 

![image](https://user-images.githubusercontent.com/29196699/166442148-04f7837a-59e9-4907-85f0-ae9abaf88fb6.png)